ga4gh/datamodel/ontologies.py

```diff
@@ -5,25 +5,38 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import fnmatch
-import os
 import ga4gh.protocol as protocol
 
 
-class FileSystemOntology(object):
+class Ontology(object):
```

Comment:
I'm confused about this @diekhans --- we have a text file that contains an Ontology (which,  at the moment corresponds to a mapping between IDs and terms), right? Which we call (say) "sequence_ontology". Where does the 'map' come into it? I'm happy to change it back, I'm just confused.